examples/nas/cream/requirements.txt

```diff
@@ -0,0 +1,12 @@
+yacs
+numpy==1.17
+opencv-python==4.0.1.24
+torchvision==0.2.1
+thop
+git+https://github.com/sovrasov/flops-counter.pytorch.git
+pillow==6.1.0
+torch==1.2
```

Comment:
Can we upgrade to latest version and pytorch and remove tensorboardx dependency?

Reply:
tensorboardX can be removed.

Reply:
Can be upgraded to the latest version, but the results can not be exactly reproduced.